keyboards/split75/config.h

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#define VENDOR_ID       0x20A0
+#define PRODUCT_ID      0x422D
```

Comment:
This VID looks like it may actually be in use, already.

Reply:
It's just copypaste, most all of the PS2AVRGB boards use this combo (I think because BMC required it for the magic bootloader jump functionality). It should definitely be changed.

Reply:
I've changed the Product ID to another value so now the combination of Vendor and Product IDs should be unique. I assume that should be correct as whatever Vendor ID was there before, should be right as it was copied from a similar board?